### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by matt-sm**
- ~common component for errors from actions~
- uses existing `ErrorBox` to display api errors
- decreases amount of error state

TODO
- needs to be moved out of user actions
- ~focus/scroll when error displayed~
- ~when to "clear" errors?~

### CODE REVIEW

File: app/components/CreateUser/CreateUser.js [hunk 1]
```diff
@@ -11,16 +12,15 @@ export class CreateUser extends Component {
...
 
     return (
       <div className="row">
         <div className="col-sm-push-2 col-sm-8 col-xs-12">
-          {loadSignupSuccess &&
-            <div>
-              <CreateUserForm initialState={userRegisterDetails} />
-            </div>}
-          {loadSignupErrored && <div>error loading user registration details</div>}
+          <article role="main">
```
Comment (marcoalfonso) [inline app/components/CreateUser/CreateUser.js]: Why are you using article? Coding html like it is 1999 :P?

Comment (matt-sm) [reply app/components/CreateUser/CreateUser.js]: I'm just copying what was already there :)

Should it be main?

File: app/actions/memberActions.js [hunk 2]
```diff
@@ -155,11 +113,22 @@ export function createUser(values) {
...
       }
       dispatch(handleDataLoading(false))
     })
   }
 }
+
+export function setErrorMessage(message) {
+  return {
+    type: SET_ERROR_MESSAGE,
+    message: message
```
Comment (Blackfry) [inline app/actions/memberActions.js]: can be simplified to `message`

File: app/components/SignupForm/SignupForm.js [hunk 3]
```diff
@@ -88,29 +86,13 @@ class SignupForm extends BaseForm {
...
-          : <li>
-              <p>
-                The Digital Marketplace encountered an error trying to send your signup email. Please try again later or{' '}
-                <a href="/contact-us" target="_blank" rel="external">
-                  {' '}contact us{' '}
-                </a>{' '}
-                for assistance.
-              </p>
-            </li>
+        signupSuccess: nextProps.signupSuccess
```
Comment (Blackfry) [inline app/components/SignupForm/SignupForm.js]: with the html removed from `signupMessage` we should be able to remove signupSuccess from component state and rely on `this.props.signupSuccess`

**Pull request merged**